packages/mds-compliance-service/tests/index.spec.ts

```diff
@@ -123,6 +123,14 @@ describe('ComplianceSnapshots Service Tests', () => {
     }
   })
 
+  it('Gets ComplianceSnapshots by TimeInterval with empty provider_ids list', async () => {
+    await expect(
+      ComplianceServiceClient.getComplianceSnapshotsByTimeInterval({
```

Comment:
Was the previous behavior causing this execution path to throw since the SQL was invalid?

Reply:
Yep